mungegithub/mungers/mungers.go

```diff
@@ -88,6 +89,7 @@ func InitializeMungers(config *github.Config, features *features.Features) error
 		if err := munger.Initialize(config, features); err != nil {
 			return err
 		}
+		glog.Infof(mungerutil.PrettyString(munger))
```

Comment:
could we just use `github.com/kr/pretty` ? maybe we don't need a whole new dep just for this. Nevermind.